IFC4x3/Constants/t/TERRAIN_2TKelKxjbD$POIgGNYWWXi.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TERRAIN_2TKelKxjbDPOIgGNYWWXi" Name="TERRAIN" UniqueId="9d528bd4-eed9-4dfd-9612-a905e282086c" Status="Deprecated" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TERRAIN_2TKelKxjbDPOIgGNYWWXi" Name="TERRAIN" UniqueId="9d528bd4-eed9-4dfd-9612-a905e282086c" />
 
```

Comment:
IfcGeoScienceModel from IfcTunnel will be covering this in another way.

Reply:
Agreed.